packages/react-core/src/components/DescriptionList/DescriptionListDescription.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import styles from '@patternfly/react-styles/css/components/DescriptionList/description-list';
+import { css } from '@patternfly/react-styles';
+
+export interface DescriptionListDescriptionProps extends React.HTMLProps<HTMLElement> {
+  /** Anything that can be rendered inside of list description */
+  children: React.ReactNode;
+  /** Additional classes added to the DescriptionListDescription */
+  className?: string;
+  /** Indicates which component will be used for the text wrapped by the description */
+  textComponent?: React.ReactNode;
+  /** Default hyperlink location */
+  href?: string;
+  /** Icon added to the left of the label text. */
+  icon?: React.ReactNode;
+}
+
+export const DescriptionListDescription: React.FunctionComponent<DescriptionListDescriptionProps> = ({
+  children = null,
+  textComponent = 'span',
+  className,
+  href,
+  icon,
+  ...props
+}: DescriptionListDescriptionProps) => {
+  const DescriptionListText: any = textComponent;
+  return (
+    <dd className={css(styles.descriptionListDescription, className)} {...props}>
+      <DescriptionListText href={href} className={'pf-c-description-list__text'}>
```

Comment:
Why are yo using the class name directly here instead of `styles`?
Why do we have the `href` here?  I don't thins the 'pf-c-description-list__text' is intended to be an `<a>`
@mcoker @mnolting.  Is the child of the `dd` supposed to be configurable or always a div (like in core)?

Reply:
I use the class name there because `styles.descriptionListText` doesn't exist -- there is no `.pf-c-description-list__text class or corresponding styles in https://github.com/patternfly/patternfly/blob/master/src/patternfly/components/DescriptionList/description-list.scss.

You're right; looks like the <a> tag should be the child of a div -- will fix.